styles.css

```diff
@@ -181,4 +704,170 @@ ul.navbar-items {
     background-size: 250px;
     background-repeat: no-repeat;
   }
+
+  /* works-section responsive style  */
+
+  #works-section {
+    margin-bottom: 100px;
+  }
+
+  .box-container {
+    flex-direction: column;
+    margin: 5px 10px;
+  }
+
+  .left-section {
+    margin-bottom: 20px;
+  }
+
+  .left-section img {
+    width: 100%;
+  }
+
+  .grid-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+    gap: 10px;
+    margin: 0;
+    padding-left: 4%;
+    padding-right: 4%;
+  }
+
+  .first-desc {
+    text-align: left;
+    margin: 10px;
+  }
+
+  .first-desc h3 {
+    font-size: 16px;
+  }
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 use the correct font family and styles to be similar to the Figma template 👇🏽 

![image](https://github.com/Xghayor/Portfolio/assets/58642949/81358cb7-df5e-4f1e-85b0-192b217f84a0)